.circleci/config.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:6.11.1
+
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
```

Comment:
Hmm, based on the same logic of the Ruby Circle config files I'm inclined to agree with you, but in the [docs for Node builds](https://circleci.com/docs/2.0/sample-config/) it uses `package.json` to watch for changed dependencies. I'm okay with either, I think.